Orange/widgets/visualize/utils/widget.py

```diff
@@ -704,10 +704,17 @@ def _manual_move_finish(self, anchor_idx, x, y):
     def _get_projection_data(self):
         if self.data is None or self.projection is None:
             return None
+
+        names = get_unique_names(self.data.domain, [a.name for a in
+                                                    self.projection.domain.attributes])
+
+        attributes = tuple([a.copy(name=n) for n, a in
+                      zip(names, self.projection.domain.attributes)])
```

Comment:
Lint is complaining about the indentation. Speaking of lint, please keep the line length up to 79 characters.